src/virtual_memory/anonymous.cpp

```diff
@@ -0,0 +1,111 @@
+/**
+ * @file
+ * @brief This file implements the virtual memory and virtual address handling
+ * @copyright Eta Scale AB. Licensed under the Eta Scale Open Source License. See the LICENSE file for details.
+ */
+
+#include <cerrno>
+#include <cstddef>
+#include <fcntl.h>
+#include <iostream>
+#include <string>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <sys/statvfs.h>
+#include <sys/types.h>
+#include <system_error>
+#include <unistd.h>
+#include "virtual_memory.hpp"
+
+/** @todo hardcoded start address */
+static const char* ARGO_START = (char*) 0x200000000000l;
```

Comment:
unnamed namespaces are recommended over static globals

http://www.comeaucomputing.com/techtalk/#nostatic

Reply:
FYI the recommendation was overturned, but the reasoning still applies to us, so we'll keep the unnamed namespaces.